tests/lifecycle/test_case_test.go

```diff
@@ -28,6 +28,10 @@ type serviceLifecycleTestCase struct {
 	testCredentials        func(credentials service.Credentials) error
 }
 
+func (s serviceLifecycleTestCase) getModule() service.Module {
```

Comment:
Do we need this function? It's not exported, so I guess only code in this same package calls it. That being the case, we may as well let would-be callers of this function access the `module` attribute of `serviceLifecycleTestCase` directly.